backend/executors/TasksDirectory.js

```diff
@@ -0,0 +1,874 @@
+/**
```

Comment:
We need to break this file down. 
Otherwise it seems that mass we tried to clean just moved here.